ui/admin/app/components/form/auth-method/password/index.hbs

```diff
@@ -10,49 +10,40 @@
   @showEditToggle={{if @model.isNew false true}}
   as |form|
 >
-  <form.input
-    @value={{@model.type}}
-    @label={{t 'form.type.label'}}
-    readonly={{true}}
-    @disabled={{true}}
-  />
+  <Hds::Form::TextInput::Field @value={{@model.type}} disabled={{true}} as |F|>
```

Comment:
One extra improvement here we could do is use the `infofield` for read-only types, similar to some other resources such as host set, host, etc.

Reply:
@DhariniJeeva ok yea I saw that in another instance.  Coming in next commit.